features/netsocket/emac-drivers/TARGET_Cypress/TARGET_PSOC6/TARGET_WHD/interface/WhdSTAInterface.h

```diff
@@ -0,0 +1,117 @@
+/* WHD implementation of NetworkInterfaceAPI
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef WHD_STA_INTERFACE_H
+#define WHD_STA_INTERFACE_H
+
+#include "mbed.h"
```

Comment:
driver should not fetch mbed.h file but instead pull what is needed

Reply:
Fixed by https://github.com/ARMmbed/mbed-os/pull/10692/commits/c115f23028c674329e859c26f4b7ad06f5d3d66c#diff-8d3ee587e2e526b518f9d8b2995fd36cL20